### PR TITLE
Fix:#1415 - Validation of dates done

### PR DESCRIPTION
**Pull request by Priyanka488**
Screenshot (172)

### CODE REVIEW

File: modules/MIPS/clinical_measures.php [hunk 1]
```diff
@@ -530,9 +424,6 @@ function Form_Validate() {
               <td style="text-align:center"><?php echo $row['excluded']; ?></td>
 <?php
             }
-
-
-
```
Comment (muarachmann) [inline modules/MIPS/clinical_measures.php]: Surely the editor stuff here

Comment (Priyanka488) [reply modules/MIPS/clinical_measures.php]: @muarachmann Do i need to make any changes ?

**Comment by muarachmann** [comment]
No it is all fine will test and approve had a busy weekend sorry for
keeping you waiting

**Comment by Priyanka488** [comment]
@muarachmann No problem : )

**Comment by aethelwulffe** [comment]
Funny.  Mua, do you know what this report does?
-In whichever case, the whitespace is cleaned up, but the LH version of this report and everything in it is now 2 years out of date.  We can merge this, but I have a feeling that Mua might be working on this himself pretty soon :)

**Comment by muarachmann** [comment]
Art it is actually an outreach stuff and the main issue was the validation.
I approved cause they depend on this for their Application process.

I know there is some stuff going on in the background was looking at that
but so far as the validation is concern i see it good for the outreachy
applicant though.

**Comment by muarachmann** [comment]
I am available so far as i am pointed to what is needed to do

**Pull request merged**